prow/gerrit/adapter/trigger.go

```diff
@@ -36,15 +40,35 @@ func messageFilter(lastUpdate time.Time, change client.ChangeInfo, presubmits []
 			continue
 		}
 
-		if !pjutil.TestAllRe.MatchString(message.Message) {
+		if pjutil.TestAllRe.MatchString(message.Message) {
```

Comment:
Do we not want to use `/ok-to-test` or similar?

Reply:
it's implicitly controlled by gerrit acl rules I think. there's not really org member concept there

Reply:
Can we plumb that through? Set the ok-to-test boolean via a different mechanism?

Reply:
I mean, not anyone can view/comment on a gerrit instance, if you are able to drop a comment, we treat you as a first class citizen already :-)

Reply:
OK, so pass in a `true` hard-coded bool to the DRY method? :dagger:

Reply:
Done. `false` IIUC so that we don't care about ok-to-test altogether.